task_processing/plugins/mesos/mesos_executor.py

```diff
@@ -71,7 +71,7 @@ def __invariant__(conf):
     retries = field(type=int,
                     factory=int,
                     mandatory=False,
-                    invariant=lambda r: (r > 0, 'retries >= 0'))
+                    invariant=lambda r: (r > 0, 'retries > 0'))
```

Comment:
this is still wrong in my opinion, see https://github.com/Yelp/task_processing/pull/80#discussion_r146798275

tl;dr since retries is not including the original run and the lowest possible value user can provide is `1`, user has no way of saying "i don't want this to be retried". invariant predicate should be changed to `r >= 0`.